ramls/inventory-hierarchy/inventory-instance-records.json

```diff
@@ -0,0 +1,557 @@
+            },
+            "holdingsStatementsForIndexes": {
+              "description": "Holdings record indexes statements",
+              "type": "array",
+              "items": [
+                {
+                  "statement": {
+                    "description": "Textual description of the holdings of indexes",
+                    "type": "string"
+                  },
+                  "note": {
+                    "description": "Note attached to a holdings statement",
+                    "type": "string"
+                  }
+                }
+              ]
+            },
+            "holdingsStatementsForSupplements": {
+              "description": "Holdings record supplements statements",
+              "type": "array",
+              "items": [
+                {
+                  "statement": {
+                    "description": "Textual description of the holdings of supplementary material",
+                    "type": "string"
+                  },
+                  "note": {
+                    "description": "Note attached to a holdings statement",
+                    "type": "string"
+                  }
+                }
+              ]
+            },
+            "copyNumber": {
+              "description": "Piece ID (usually barcode) for systems that do not use holdings record",
+              "type": "string"
+            },
+            "numberOfItems": {
+              "description": "Amount of items of holdings record",
+              "type": "string"
+            },
+            "receivingHistory": {
+              "description": "Receiving history of holdings record",
+              "type": "object",
+              "properties": {
+                "entries": {
+                  "description": "Entries of receiving history",
+                  "type": "array",
+                  "items": [
+                      {
+                      "publicDisplay": {
+                        "description": "Defines if the receiving history should be visible to the public",
+                        "type": "boolean"
+                      },
+                      "enumeration": {
+                        "description": "This is the volume/issue number (e.g. v.71:no.6-2)",
+                        "type": "string"
+                      },
+                      "chronology": {
+                        "description": "Repeated element from Receiving history - Enumeration AND Receiving history - Chronology",
+                        "type": "string"
+                      }
+                    }
+                  ]
+                }
+              }
+            },
+            "tags": {
+              "description": "Arbitrary tags associated with this holding",
+              "type": "object",
+              "properties": {
+                "tagList": {
+                  "description": "List of tags",
+                  "type": "array",
+                  "items": [
+                    {
```

Comment:
This fails the build and should be changed to

              "items": {
                "type": "object",
                "properties": {

Reply:
You are right, it looks like I was wrong defined arrays of objects for some entries. I'll fix it, thanks.